web/src/invite.ts

```diff
@@ -451,10 +411,65 @@ function open_invite_user_modal(e: JQuery.ClickEvent<Document, undefined>): void
         };
 
         $("#invite-user-form .setup-tips-container").html(render_invite_tips_banner(context));
+
+        const toggler = components.toggle({
+            html_class: "invite_users_option_tabs",
+            selected: 0,
+            child_wants_focus: true,
+            values: [
+                {label: $t({defaultMessage: "Send invite email"}), key: "invite-email-tab"},
+                {label: $t({defaultMessage: "Create invite link"}), key: "invite-link-tab"},
+            ],
+            callback(_name, key) {
+                switch (key) {
+                    case "invite-email-tab":
+                        $("#invitee_emails_container").show();
+                        $("#invite-user-modal .dialog_submit_button").text(
+                            $t({defaultMessage: "Invite"}),
+                        );
+                        $("#invite-user-modal .dialog_submit_button").data(
+                            "loading-text",
+                            $t({defaultMessage: "Inviting..."}),
+                        );
+                        toggle_invite_submit_button();
+                        break;
+                    case "invite-link-tab":
+                        $("#invitee_emails_container").hide();
+                        $("#invite-user-modal .dialog_submit_button").text(
+                            $t({defaultMessage: "Generate invite link"}),
+                        );
+                        $("#invite-user-modal .dialog_submit_button").data(
+                            "loading-text",
+                            $t({defaultMessage: "Generating link..."}),
+                        );
+                        $("#invite-user-modal .dialog_submit_button").prop("disabled", false);
+                        break;
+                }
+                reset_error_messages();
+            },
+        });
+        const $container = $("#invite_users_option_tabs_container");
+        if (!settings_data.user_can_invite_users_by_email()) {
+            toggler.disable_tab("invite-email-tab");
+            $container.addClass("no_email_invites");
+            toggler.goto("invite-link-tab");
+        }
+        if (!settings_data.user_can_create_multiuse_invite()) {
+            toggler.disable_tab("invite-link-tab");
+            $container.addClass("no_link_invites");
+        }
+        const $elem = toggler.get();
+        $elem.addClass("large allow-overflow");
```

Comment:
I think you can just include these in the `html_class` parameter in the toggler constructor.